### PR TITLE
Fix openssl

### LINKED ISSUE #49: Problems with openssl
In the RSP image in particular

### PR DESCRIPTION
**Pull request by colearendt**
Close #49 

adding `/opt/python/x.x.x/bin/` to the path overwrites the system `openssl`, which can be problematic for certificate verification, updates, etc.

### CODE REVIEW

File: server-pro/Dockerfile [hunk 1]
```diff
@@ -30,11 +30,8 @@ RUN curl -o /tmp/miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-4.5
...
     rm /tmp/miniconda.sh && \
     /opt/python/${PYTHON_VERSION}/bin/conda clean -tipsy && \
-    ln -s /opt/python/${PYTHON_VERSION}/etc/profile.d/conda.sh /etc/profile.d/conda.sh && \
-    echo "PATH=/opt/python/${PYTHON_VERSION}/bin:$PATH" > /etc/profile.d/python.sh && \
     /opt/python/${PYTHON_VERSION}/bin/conda clean -a && \
     /opt/python/${PYTHON_VERSION}/bin/pip install virtualenv
-ENV PATH /opt/python/${PYTHON_VERSION}/bin:$PATH
 
 # Install other Python PyPi packages
 RUN pip install --no-cache-dir \
```
Comment (danielfrg) [inline server-pro/Dockerfile:37]: Need to use the full `/opt/python/...` here since we are removing the `PATH` above

File: server-pro/Dockerfile [hunk 2]
```diff
@@ -44,6 +41,14 @@ RUN pip install --no-cache-dir \
                 rsp_jupyter==1.2.5003 \
                 rsconnect_jupyter==1.3.3.1
 
+# Add binaries to the PATH
```
Comment (danielfrg) [inline server-pro/Dockerfile:44]: I think I will remove this all together, it does add this but other ones like jupyter will be missing and this list is just gonna get outdated.

File: server-pro/Dockerfile [hunk 3]
```diff
@@ -44,6 +41,14 @@ RUN pip install --no-cache-dir \
...
+RUN \
+    ln -s /opt/python/${PYTHON_VERSION}/bin/python /usr/local/bin/python && \
+    ln -s /opt/python/${PYTHON_VERSION}/bin/python3 /usr/local/bin/python3 && \
+    ln -s /opt/python/${PYTHON_VERSION}/bin/rsconnect /usr/local/bin/rsconnect && \
+    ln -s /opt/python/${PYTHON_VERSION}/bin/virtualenv /usr/local/bin/virtualenv && \
+    ln -s /opt/python/${PYTHON_VERSION}/bin/conda /usr/local/bin/conda
+
 # Install Jupyter extensions
 RUN jupyter-nbextension install --sys-prefix --py rsp_jupyter && \
```
Comment (danielfrg) [inline server-pro/Dockerfile:53]: Same `PATH` comment as for `pip`

**Comment by colearendt** [comment]
I must be doing something wrong in `launcher-env` because the `PATH` modification does not seem to be doing anything... 🤷‍♂️

**Comment by danielfrg** [comment]
lgtm!

**Pull request merged**